.github/workflows/continuous-integration.yml

```
@@ -34,19 +34,20 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
+      - uses: Yuri6037/Action-FakeTTY@v1.1
       - name: Update NPM
+        if: matrix.node-version <= 14
         run: npm install -g npm
```

Just to make sure a semver major NPM update  doesn't suddenly break the pipeline:

```suggestion
        run: npm install --global npm@7
```

See: https://github.com/loopbackio/cicd/issues/6

---

Well thought out, I like it.

NPM v7 was quite unstable in its early versions so I'm going to use the latest-7 tag.